lib/util/ldaputil.js

```diff
@@ -44,20 +44,16 @@ module.exports.login = function(options, username, password) {
 
   function tryFind(client) {
     var resolver = Promise.defer()
+    var filter = `&(objectClass=${options.search.objectClass})` +
+                 `(${options.search.field}=${username})`
```

Comment:
Using string filters with user-defined content is vulnerable to [LDAP Injection](https://www.owasp.org/index.php/LDAP_injection). Please use the filter builders instead.